Automattic/wp-calypso
# Migrate checklist styles to webpack

#### Changes proposed in this Pull Request

- Import `Checklist` style with webpack. Details on https://github.com/Automattic/wp-calypso/issues/27515

#### Testing instructions

- Visit `/devdocs/design/checklist` on the live branch available below
- Ensure the design/visual appearance looks the same as this page's view on `master` branch / before this PR

#### Notes

- Most of the stylesheet declarations are cross-used by multiple files, which is why I haven't broken the stylesheet individual multiple smaller files. 
- I don't see any unused styles, but if there are any I missed, happy to be pointed at them!
